jujugui/static/gui/src/app/components/header-help/header-help.js

```diff
@@ -0,0 +1,111 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('header-help', function() {
+
+  const HeaderHelp = React.createClass({
+
+    propTypes: {
+      appState: React.PropTypes.object.isRequired
+    },
+
+    getInitialState: function() {
+      return {
+        showHelpMenu: false
+      };
+    },
+
+    handleClickOutside: function(e) {
+      this.setState({showHelpMenu: false});
+    },
+
+    toggleHelpMenu: function() {
+      this.setState({ showHelpMenu: !this.state.showHelpMenu });
+    },
+
+    _handleDocsClick: function() {
+      window.open('https://jujucharms.com/docs/stable/getting-started');
```

Comment:
This will be blocked by browsers - If you used an anchor tag instead then it will open as expected.